x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -416,11 +426,24 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
             features: [FEATURE.casesRead],
           },
           [SecurityPageName.caseConfigure]: {
-            features: [FEATURE.casesCrud],
+            features: [
+              FEATURE.casesCreate,
+              FEATURE.casesRead,
+              FEATURE.casesUpdate,
+              FEATURE.casesDelete,
+              FEATURE.casesPush,
+            ],
             isPremium: true,
           },
+          // TODO: can we just use create here?
           [SecurityPageName.caseCreate]: {
-            features: [FEATURE.casesCrud],
+            features: [
```

Comment:
@cnasikas  We had `crud` required here before should it really just be `create`?

Reply:
Currently, this link is configured to need all the cases permissions, from what you commented, shouldn't it be
`features: FEATURE.casesCreate`?

Reply:
We decided to leave as it is for the moment to be more conservative. If and when we introduce more sub-features we will consider them again.